src/Raven.Server/Documents/PeriodicBackup/PeriodicBackupRunner.cs

```diff
@@ -320,6 +323,68 @@ public long StartBackupTask(long taskId, bool isFullBackup)
             return CreateBackupTask(periodicBackup, isFullBackup, SystemTime.UtcNow);
         }
 
+        public async Task DelayAsync(long taskId, TimeSpan delay, X509Certificate2 clientCert)
+        {
+            foreach (var periodicBackup in _periodicBackups)
+            {
+                var runningTask = periodicBackup.Value.RunningTask;
+                if (runningTask == null || runningTask.Id != taskId)
+                    continue;
+
+                var delayUntil = DateTime.UtcNow.AddTicks(delay.Ticks);
+
+                var command = new DelayBackupCommand(_database.Name, RaftIdGenerator.NewId())
+                {
+                    TaskId = periodicBackup.Key,
+                    DelayUntil = delayUntil
+                };
+
+                try
+                {
+                    (long index, _) = await _database.ServerStore.SendToLeaderAsync(command);
+                    await _database.ServerStore.WaitForCommitIndexChange(RachisConsensus.CommitIndexModification.GreaterOrEqual, index);
+                }
+                catch (Exception e)
+                {
+                    if (_logger.IsOperationsEnabled)
+                    {
+                        var msg =
+                            $"Fail to delay backup task with task id '{taskId}' cluster-wide, the task was delayed until '{delayUntil}' UTC only on the current node.";
+                        
+                        _logger.Operations(msg, e);
+                    }
+                }
+
+                if (_auditLog.IsInfoEnabled)
+                {
+                    var msg = $"Backup task with task id '{taskId}' was delayed until '{delayUntil}' UTC";
+
+                    if (clientCert != null)
+                        msg += $" by {clientCert.Subject} ({clientCert.Thumbprint})";
+
+                    _auditLog.Info(msg);
+                }
+
+                periodicBackup.Value.BackupStatus.DelayUntil = delayUntil;
+
+                _forTestingPurposes?.OnBackupTaskRunHoldBackupExecution?.SetResult(null);
+                _database.Operations.KillOperation(taskId);
+                
+                try
+                {
+                    await runningTask.Task; // wait for the running task to complete
+                }
+                catch
+                {
+                    // task has ended, nothing we can do here
+                }
+                
+                return;
+            }
+
+            throw new ArgumentException($"Backup task {taskId} isn't registered");
```

Comment:
InvalidOperationException

Reply:
The message is not clear, extend it so user will understand what is going on

Reply:
Done.